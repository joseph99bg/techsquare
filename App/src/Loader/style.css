.loader-holder {
    text-align: center;
}