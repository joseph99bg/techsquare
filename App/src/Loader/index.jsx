import React from 'react'
import './style.css'

const Loader = () => {
    return (
        <div className="loader-holder">
            <img src="/images/loader.gif" alt=""/>
        </div>
    )
}

export default Loader