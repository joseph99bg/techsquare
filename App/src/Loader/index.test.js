import React from 'react'
import Loader from './index'
import renderer from 'react-test-renderer'

describe('Loader component', () => {
    test('should render correctly', () => {
        const component = renderer.create(
            <Loader />
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})