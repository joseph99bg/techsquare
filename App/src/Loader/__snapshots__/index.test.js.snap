// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should render correctly 1`] = `
<div
  className="loader-holder"
>
  <img
    alt=""
    src="/images/loader.gif"
  />
</div>
`;
